### PR TITLE
Frontend portal pagination 

### PR DESCRIPTION
**Pull request by abdullai-t**
####  Summary / Highlights
This PR contains changes as a result of data pagination in the API. Actions, testimonials, and teams will be loaded to the frontend in chunks( 50 items). 
#### Details (Give details about what this PR accomplishes, include any screenshots, etc)

https://user-images.githubusercontent.com/42780120/194958445-02001292-9d15-4024-83c1-d40ed6d99ad9.mp4

#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [x] I've given my PR a meaningful title.
* [ ] I linked this PR to the relevant issue.
* [ ] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

File: src/components/Pages/ProfilePage/ProfilePage.js [hunk 1]
```diff
@@ -102,6 +102,7 @@ class ProfilePage extends React.Component {
   }
   showCommunitiesSection() {
     const { user } = this.props;
+       console.log('===== ToLog USER========', user);
```
Comment (frimpongopoku) [inline src/components/Pages/ProfilePage/ProfilePage.js:105]: Remove

File: src/components/Pages/ProfilePage/ProfilePage.js [hunk 2]
```diff
@@ -515,13 +516,15 @@ class ProfilePage extends React.Component {
     );
   };
   renderCommunities(communities) {
+       console.log('===== ToLog commms ========', communities);
```
Comment (frimpongopoku) [inline src/components/Pages/ProfilePage/ProfilePage.js:519]: Remove

File: src/components/Pages/ProfilePage/ProfilePage.js [hunk 3]
```diff
@@ -515,13 +516,15 @@ class ProfilePage extends React.Component {
...
     if (!communities)
       return (
         <div style={{ textAlign: "center" }}>
           <METextView>You haven't joined any communities yet</METextView>
         </div>
       );
     return Object.keys(communities).map((key) => {
+      console.log('===== ToLog ========', communities);
```
Comment (frimpongopoku) [inline src/components/Pages/ProfilePage/ProfilePage.js:527]: Remove

**Review by frimpongopoku — CHANGES_REQUESTED** [review]
LGTM

**Comment by Davidjwkinghorn** [comment]
When I go to test this I get a compilation error. 
"""Failed to compile.

./src/components/Utils.js
  Line 319:3:  Expected an assignment or function call and instead saw an expression  no-unused-expressions

Search for the keywords to learn more about each error."""

**Pull request closed without merging**